src/app/containers/SignupContainer/SignupContainer.js

```diff
@@ -171,24 +171,26 @@ const SignupContainer = ({ match, history, onLogin, stopRedirect }) => {
         </div>
     );
 
+    const isBundlePlan = selectedPlan.planName === PLAN.BUNDLE_PLUS;
+    const homepageUrl = isBundlePlan ? 'https://protonvpn.com/blackfriday' : 'https://protonvpn.com';
```

Comment:
Instead of detecting the bundle plans, what do you think about detecting the Black Friday coupon since the condition is changing the link to `https://protonvpn.com/blackfriday`?